class_ids: {
  "klt": 6,
  "power_drill": 2,
}
